services/fluxmonitor_test.go

```diff
@@ -90,15 +93,16 @@ func TestNewFluxMonitor(t *testing.T) {
 		absoluteThreshold float64
 		heartbeat         time.Duration
 		pollInterval      time.Duration
-		want              string
+		// multiply          int32
```

Comment:
Should we remove this here?

Reply:
I left it because I was thinking to add some case for multiply as well. But can remove for now.